# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-counter-styles/myanmar/css3-counter-styles-141.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/css-counter-styles/myanmar/css3-counter-styles-141.html.ini
--- a/testing/web-platform/meta/css/css-counter-styles/myanmar/css3-counter-styles-141.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/css-counter-styles/myanmar/css3-counter-styles-141.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,9 +1,9 @@
 [css3-counter-styles-141.html]
   expected:
-    if not webrender and (os == "linux") and debug: ["PASS", "FAIL"]
+    if not webrender and (os == "linux") and debug: ["FAIL", "PASS"]
     if not webrender and (os == "linux") and not debug: ["PASS", "FAIL"]
+    if webrender and (os == "linux"): ["FAIL", "PASS"]
     if not webrender and (os == "android"): PASS
     if not webrender and (os == "mac"): PASS
     if not webrender and (os == "win"): PASS
-    if webrender and (os == "linux"): ["FAIL", "PASS"]
     FAIL